### PR TITLE
Add missing picture to the platform whitepaper Chinese version

### PR DESCRIPTION
**Pull request by node**
The image translation work is shared at https://docs.google.com/presentation/d/16AeFfd8JG5aJYuTYPn30zNCKiuAnc7hEheRLQwGWHH8/edit?usp=sharing

Refer PR #623

### CODE REVIEW

File: website/content/zh/wgs/platforms/whitepaper/assets/.empty
Comment (lianmakesthings) [inline website/content/zh/wgs/platforms/whitepaper/assets/.empty:1]: This file should be removed

**Review by lianmakesthings — CHANGES_REQUESTED** [review]
LGTM except for that empty file

**Comment by node** [comment]
Sincerely invite @rootsongjc  to help review this image if he is free.  😄

**Review by roberthstrand — APPROVED** [review]

**Comment by roberthstrand** [comment]
While it looks technically correct, I will post this is the CNCF Ambassador channel to see if any native speakers could give it a once-over before merging.

**Comment by node** [comment]
Update: 
Add drawio raw file in Chinese while update PNG file .    Many thanks to @addozhang for your advice and help.

**Comment by roberthstrand** [comment]
@node, Could you take a look at the comments left by @tao12345666333? I can't really judge whether one word is right or the other, so this is something that I need your help on.

**Comment by node** [comment]
> @node, Could you take a look at the comments left by @tao12345666333? I can't really judge whether one word is right or the other, so this is something that I need your help on.

Fixed all of them. Please check it again @tao12345666333 , and thanks for your review.

**Review by roberthstrand — APPROVED** [review]

**Pull request merged**